src/pages/AdminDashboard/AdminDashboard.jsx

```diff
@@ -1,16 +1,57 @@
 import React from 'react';
+import { ChakraProvider, Button } from '@chakra-ui/react';
+
 import Map from '../../components/Map/Map';
 import BoxApproval from '../../components/BoxApproval/BoxApproval';
+import NavBar from '../../components/NavBar/NavBar';
 import './AdminDashboard.css';
 
 function AdminDashboard() {
+  const sideBarComponent = document.getElementsByClassName('side-bar')[0];
+  const mapComponent = document.getElementsByClassName('map')[0];
+  const reviewSubmissionComponent = document.getElementsByClassName('review-submission-button')[0];
+
+  const setReviewSubmission = () => {
+    sideBarComponent.style.display = 'block';
```

Comment:
I still used display for sideBarComponent because visibility still allocates space for the component even when it is hidden, therefore would cause a white section on the page.

Reply:
Okay, that's fine